Resources/Prototypes/_Moffstation/Loadouts/loadout_groups.yml

```diff
@@ -452,27 +461,20 @@
   - InmatePrisonerPDA
   - TestSubjectPDA
 
-# Department Heads can use any department Bag
+# Reporter
 - type: loadoutGroup
-  id: ChiefEngineerBackpack
```

Comment:
Dangerous to merge just this on its own.
edit: er wait, I'm just blind, It seems. Thought this was straight up gone here but added in another PR